ui-lib/src/main/res/ui/advanced_settings/values/strings.xml

```diff
@@ -1,11 +1,10 @@
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="advanced_settings_backup_wallet">Recovery phrase</string>
-    <string name="advanced_settings_export_private_data">Export private data</string>
-    <string name="advanced_settings_choose_server">Choose a server</string>
+    <string name="advanced_settings_title">Advanced Settings</string>
+    <string name="advanced_settings_recovery">Recovery Phrase</string>
+    <string name="advanced_settings_export">Export Private Data</string>
+    <string name="advanced_settings_choose_server">Choose a Server</string>
     <string name="advanced_settings_currency_conversion">Currency Conversion</string>
-
-    <string name="advanced_settings_delete_wallet">
-        Delete <xliff:g id="app_name" example="Zashi">%1$s</xliff:g>
-    </string>
-    <string name="advanced_settings_delete_wallet_footnote">(You will be asked to confirm on next screen)</string>
+    <string name="advanced_settings_coinbase">Buy ZEC with Coinbase</string>
```

Comment:
We can change the fixed currency to variable as part of the following PR https://github.com/Electric-Coin-Company/zashi-android/pull/1542